resources/views/exhibitions/components/related-events.blade.php

```diff
@@ -44,6 +44,9 @@
                     <div class="related-event-wrapper">
                         @if(!empty($event['heading']))
                             <h4 class="cta-title">{{ $event['heading'] }}</h4>
+                            @if(!empty($event['event_date']))
```

Comment:
I would personally recommend moving this outside of the `@if(!empty($event['heading']))` loop (e.g. there may be no heading but there might be an event date), but this is a really minor thing and probably won't cause any problems.

Reply:
Thanks for this. I get this template updated.